x/ibc/04-channel/keeper/keeper.go

```diff
@@ -125,6 +125,23 @@ func (k Keeper) SetNextSequenceAck(ctx sdk.Context, portID, channelID string, se
 	store.Set(host.KeyNextSequenceAck(portID, channelID), bz)
 }
 
+// GetPacketReceipt gets a packet receipt from the store
+func (k Keeper) GetPacketReceipt(ctx sdk.Context, portID, channelID string, sequence uint64) (string, bool) {
+	store := ctx.KVStore(k.storeKey)
+	bz := store.Get(host.KeyPacketReceipt(portID, channelID, sequence))
+	if bz == nil {
+		return "", false
+	}
+
+	return string(bz), true
+}
+
+// SetPacketReceipt sets an empty packet receipt to the store
```

Comment:
maybe we should explain why it is empty?

Reply:
that's how it described on was on the spec (empty string)

Reply:
we just need to set the key, I guess we could also set it to null but that might not be supported in all stores